src/classes/ADDR_IValidator.cls

```diff
@@ -35,19 +35,43 @@
 */
 public Interface ADDR_IValidator {
 
-    /** Return the service name to be displayed in the drop-down list. **/
+    /** 
+    * @description returns the service name to be displayed in the drop-down list. 
+    * @return string
+    */
     String getServiceName();
     
-    /** Return the default endpoint URL. **/
+    /** 
+    * @description returns the default endpoint URL. 
+    * @return string
+    */
     String getDefaultURL();
     
-    /** Return true if batch address validation is supported. If not, the batch address validation button
-    will be disabled. **/
+    /** 
+    * @description returns the help text to be shown when the user selects the service from the drop-down. It
+    * may specify things such as the required fields. 
+    * @return string
+    */
+    String getHelpText();
```

Comment:
Sounds like I need to do another pull from dev. My problem was I couldn't
even get the build going on that branch manually. I hadn't heard from Jason
that he fixed anything. Maybe another intermittent issue, or he forgot to
tell me it was fixed.  Thx.



On Jan 25, 2015, at 10:40 AM, Kevin Bromer <notifications@github.com> wrote:

In src/classes/ADDR_IValidator.cls
<https://github.com/SalesforceFoundation/Cumulus/pull/1282#discussion_r23506370>
:

>      String getDefaultURL();
>
> -    /** Return true if batch address validation is supported. If not, the batch address validation button
> -    will be disabled. **/
> +    /**
> +    * @description returns the help text to be shown when the user selects the service from the drop-down. It
> +    * may specify things such as the required fields.
> +    * @return string
> +    */
> +    String getHelpText();

Here, this is new - so it need to be implemented by everything that
implements this ADDR_IValidator interface

—
Reply to this email directly or view it on GitHub
<https://github.com/SalesforceFoundation/Cumulus/pull/1282/files#r23506370>.